### DISCUSSION TITLE
Option to disable autoplay

### DISCUSSION THREAD

**Discussion by neeleysc** [PaulWoitaschek/Voice#2646] [Ideas]
Is there a way we can have an option which will disable autoplay after connecting to a Bluetooth device? Voice still likes to autoplay when my phone connects to the car Bluetooth.

**Comment by devmarxx** [comment]
I would like to have this feature too.

I now have a system wide solution with "Automate":

https://llamalab.com/automate/community/flows/25870

This works well, but on some rare occasions the car still finds a way to autoplay. So it would be great to have an option in Voice to stop autoplay directly.

Thanx for the great software :-)

Regards

devmarxx
